ganga/GangaDirac/Lib/Server/DiracCommands.py

```diff
@@ -405,3 +404,26 @@ def checkSEStatus(se, access = 'Write'):
     '''
     result = dirac.checkSEAccess(se, access)
     return result
+
+@diracCommand
+def getDDBtags( lfn ):
+    ''' returns the DDB and CONDDB tags of the last 
+	production step for a given LFN 
+    '''
+    from LHCbDIRAC.BookkeepingSystem.Client.BookkeepingClient import BookkeepingClient
+    import types
+    bk = BookkeepingClient()
+    prod = long(lfn.split('/')[5]) # not sure if this works in all cases 
```

Comment:
It might be a better idea to move this to outside the function and pass the production ID as an argument. You can then check that you get the same DB tags for all production IDs.